apps/site/pages/docs/migration-guides/Migrating to 2.2.65 - graphql-utils depreciation.mdx

```diff
@@ -0,0 +1,111 @@
+# Migrating to 2.2.65 - graphql-utils depreciation
+
+From 2.2.60 on, the `@faststore/graphql-utils` is officially deprecated. It was previously used internally by `@faststore/core` to handle GraphQL optimizations and GraphQL definitions.
```

Comment:
```suggestion
In this guide, learn how to migrate your store version to v2.2.65 to leverage the latest improvements in the API extension.

Version 2.2.65 and above, introduces the following enhancements to API extension users: 

- Deprecation of the `@faststore/graphql-utils` package in favor of the [`client-preset`](https://the-guild.dev/graphql/codegen/plugins/presets/preset-client) plugin.

- Refinement `gql` helper usage for a smoother and more efficient GraphQL query handling.

- Enhancement security measures to a more secure method for invoking queries and mutations to safeguard your store's data.

- Optimization call processes for queries or mutations defined within `@faststore/core`.

import { Callout } from 'nextra/components'
 
<Callout type="info" emoji="ℹ️">
  For more details about these changes, also refer to the release note [TBD](/tbd)
</Callout>
```

Reply:
The guide is important to have, but it appears that a release note is also necessary. This note will officially inform CX about the migration and enhance visibility for the guide. I'm currently drafting the release note [here](https://docs.google.com/document/d/1HZINqZ0f-iDQb3CnuyPki2A74RmJ0Rbqr4Ud5S1Fi6Q/edit?usp=sharing), so the team can also review it.

Reply:
I agree! Thanks for that.